openapi/paths/storefront/kyc-liveness-sessions@{id}@finish.yaml

```
@@ -8,12 +8,17 @@ post:
   summary: Finish KYC liveness session
   operationId: StorefrontPostKycLivenessSessionFinish
   x-sdk-operation-name: finish
-  description: Attempt to finalize a pending KYC liveness session.
+  description: |-
+    Attempt to finalize a pending KYC liveness session.
+    Session will be finalized only once the KYC liveness session in the third-party service is in a final state.
```

suggestion: for readability
```suggestion
    The session is finalized when the KYC liveness session in the third-party service is in a final state.
```